crates/uv-auth/src/lib.rs

```diff
@@ -8,6 +10,12 @@ pub use store::AuthenticationStore;
 
 use url::Url;
 
+// TODO(zanieb): Consider passing a store explicitly throughout
+lazy_static! {
+    // Global authentication store for a `uv` invocation
+    pub static ref GLOBAL_AUTH_STORE: AuthenticationStore = AuthenticationStore::default();
```

Comment:
Is it possible to do this with `Lazy` instead of `lazy_static`, since we already had that dep elsewhere? (I know `lazy_static!` was introduced in the prior PR.)

Reply:
Yeah I think so

Reply:
Done